sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -151,7 +155,7 @@ export interface JsonWebKey {
 
 // @public
 export class KeyClient {
-    constructor(vaultUrl: string, credential: TokenCredential, pipelineOptions?: PipelineOptions);
+    constructor(vaultUrl: string, credential: TokenCredential, pipelineOptions?: KeyClientOptions);
```

Comment:
Might make sense to call the parameter `clientOptions` on all the clients.

Reply:
We've been avoiding having the same class names on the different clients since it could be confusing for people to use them. I'll wait and see what other people say for this feedback, if you don't mind 🌞

Reply:
KeyClientOptions goes into a KeyClient. This is the naming convention set forth in the Azure SDK guidelines.